org.eclipse.buildship.core/build.gradle

```diff
@@ -4,4 +4,8 @@ apply plugin: eclipsebuild.BundlePlugin
 tasks.processResources.doFirst {
     def buildshipRunimeProperties = file('src/main/resources/org/eclipse/buildship/core/internal/gradle/buildship-runtime.properties')
     buildshipRunimeProperties.text = "ToolingApiVersion=$toolingApiVersion"
+}
+
+dependencies {
+    compile 'org.slf4j:slf4j-api:1.7.2'
```

Comment:
should this be implementation or api?

Reply:
The same as the tooling-api requires: slf4j-api.

Reply:
Sorry, I meant the configuration.  Instead of compile, should it be the api or implementation configuration?

Reply:
Ah, that was a silly comment from me. 
So, the build script predates api-implementation separation and simply [uses](https://github.com/eclipse/buildship/blob/86fe8c98417d38bc421c02578ab06685f3f0da79/buildSrc/src/main/groovy/eclipsebuild/BundlePlugin.groovy#L141) the `compile` scope. Improving that is out of scope at this point.